x/mongo/driver/topology/server.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) MongoDB, Inc. 2017-present.
+// Copyright (C) MongoDB, Inc. 2023-present.
```

Comment:
Do we constantly update the copyright notice whenever we update the source code?

Reply:
I've reverted this back.